Frontend/ui-driver/src/Flow.purs

```diff
@@ -452,6 +452,7 @@ onBoardingFlow = do
                         else if resp.dlVerificationStatus == "LIMIT_EXCEED" then Just "DL" 
                         else Nothing
       cityConfig = getCityConfig config.cityConfig (getValueToLocalStore DRIVER_LOCATION)
+      referralCodeAdded = if getValueToLocalStore REFERRAL_CODE_ADDED == "true" then true else false
```

Comment:
`      referralCodeAdded = getValueToLocalStore REFERRAL_CODE_ADDED == "true"`

Replace it with this